SmartTutor/KnowledgeComponentModel/KnowledgeComponentService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using SmartTutor.KnowledgeComponentModel.KnowledgeComponents;
+using SmartTutor.KnowledgeComponentModel.KnowledgeComponents.Repository;
+
+namespace SmartTutor.KnowledgeComponentModel
+{
+    public class KnowledgeComponentService : IKnowledgeComponentService
+    {
+        private readonly IKnowledgeComponentRepository _knowledgeComponentRepository;
+
+        public KnowledgeComponentService(IKnowledgeComponentRepository knowledgeComponentRepository)
+        {
+            _knowledgeComponentRepository = knowledgeComponentRepository;
+        }
+
+        public List<KnowledgeComponent> GetKnowledgeComponents()
```

Comment:
After the major refactoring, we will most likely move to using https://github.com/altmann/FluentResults. No action items at this point, other than to familiarize yourself with the concept.

Reply:
Noted.